packages/validation/test/fixtures/br/_module.wast

```diff
@@ -0,0 +1,25 @@
+(module
```

Comment:
this test is disabled right?

Reply:
Damn I forgot to delete this one, sorry! Good catch